docs/src/getting-started/scarb.md

```diff
@@ -8,7 +8,7 @@ Starknet Foundry uses [Scarb](https://docs.swmansion.com/scarb) to:
 - [build contracts](https://docs.swmansion.com/scarb/docs/extensions/starknet/contract-target.html)
 
 One of the core concepts of Scarb is its [manifest file](https://docs.swmansion.com/scarb/docs/reference/manifest.html) - `Scarb.toml`.
-It can be also used to provide [configuration](../projects/configuration.md) for Starknet Foundry.
+It can be also used to provide [configuration](../projects/configuration.md) for Starknet Foundry Forge.
```

Comment:
```suggestion
It can be also used to provide [configuration](../projects/configuration.md) for Starknet Foundry.
```
I believe we need to make more effort describing the difference between foundry and scarb tomls.

Good example is where configuration should be taken from (which package) for `sncast get-nonce`

Reply:
> Not sure what should be done to make it more explicit to be honest. In configuration part it is described that forge uses Scarb.toml and sncast uses snfoundry.toml, I guess it is a good place to describe such things.

I believe sncast may use Scarb.toml in the future, and forge snfoundry.toml . I believe the difference is not in the tool but between semantics of those two configs. Scarb.toml is a package definition. snfoundry.toml holds configuration that doesn't have to be tied to a specific package.

Reply:
> Scarb.toml is a package definition. snfoundry.toml holds configuration that doesn't have to be tied to a specific package.

That is true, but only for sncast. Correct me if I'm wrong, but forge can technically hold (at least at the moment) some forge-specific things in Scarb.toml as well, right?

Reply:
Yes, because everything in forge is package specific, as every test belongs to specific package, so it is possible to always resolve which config should be the source. 

get_nonce is a good example why snfoundry.toml is necessary in cast, because from which package you would take the config otherwise.

Cast still should use scarb toml in case we have things that can always be resolved in the context of specific package. I think for example that script configuration should be held (or inherited) by scarb toml from snfoundry toml as scripts are always in the context of specific package.

Reply:
Ok, so my understanding is that I should:
- drop `Forge` from the description about holding config in Scarb.toml
- emphasize more than snfoundry is used for cast, because we are not always in some package (as opposed to forge)

is that right?

> Cast still should use scarb toml in case we have things that can always be resolved in the context of specific package. I think for example that script configuration should be held (or inherited) by scarb toml from snfoundry toml as scripts are always in the context of specific package.

I think this might just make things more messy. I'd stick with using snfoundry for that, as technically we do not configure a package, but rather a script execution (so cast). + actually support for reading configs from Scarb.toml is dropped for cast, so it might be misleading to suggest something like this is possible

Reply:
I am ok with it but there is no explanation (or at least I don't see it) why snfoundry toml is used instead of Scarb toml for cast.

Reply:
sure, done https://github.com/foundry-rs/starknet-foundry/pull/1647/commits/1cd4ef0544e5c279ec7419d08dc4f6485e446605